README.md

```diff
@@ -2,12 +2,15 @@
 
 [![build status](https://github.com/Scewo/python-dotnetinteropt/actions/workflows/format-lint.yml/badge.svg)](https://github.com/Scewo/python-dotnetinteropt/actions/workflows/format-lint.yml)
 [![build status](https://github.com/Scewo/python-dotnetinteropt/actions/workflows/generate-doc.yml/badge.svg)](https://github.com/Scewo/python-dotnetinteropt/actions/workflows/generate-doc.yml)
-[![build status](https://github.com/Scewo/python-dotnetinteropt/actions/workflows/unit-tests.yml/badge.svg)](https://github.com/Scewo/python-dotnetinteropt/actions/workflows/unit-tests.yml)
 
 This package is a tool to easily install NuGet packages and load the compiled
 .NET DLL's into python. It uses [Pythonnet](https://pythonnet.github.io/) in the
 background to load and run the DLL's.
 
+In order to use this package you need to have [.NET](https://dotnet.microsoft.com/en-us/)
+and [NuGet](https://learn.microsoft.com/en-us/nuget/install-nuget-client-tools)
```

Comment:
Why is Nuget explicitly required? 🤔

Reply:
Ah maybe that's not needed you are right.